1-exercises/A-accessing-values/exercise3.js

```diff
@@ -21,12 +21,15 @@ let basketballTeam = {
 */
 
 // write code here
-
-
+function topPlayers(player) {
```

Comment:
This works well. "team" would be a better variable name here, rather than "player".